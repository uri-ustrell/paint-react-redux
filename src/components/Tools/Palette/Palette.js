import React from "react";

const Palette = () => <div className="paint-palette"></div>;

export default Palette;
