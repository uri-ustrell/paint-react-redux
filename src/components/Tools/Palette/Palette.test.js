import React from "react";
import { shallow } from "enzyme";
import Palette from "./Palette";

describe("Palette component", () => {
	let wrapper;

	beforeEach(() => {
		wrapper = shallow(<Palette />);
	});

	it("must exists", () => {
		expect(wrapper.find("div.paint-palette").length).toBe(1);
	});

	it("should render correctly", () => {
		expect(wrapper).toMatchSnapshot();
	});
});
