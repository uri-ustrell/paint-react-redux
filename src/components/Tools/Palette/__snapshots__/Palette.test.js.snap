// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Palette component should render correctly 1`] = `
<div
  className="paint-palette"
/>
`;
